### DISCUSSION TITLE
Rendering animated county choropleth - Error 'requested entity too large'

### DISCUSSION THREAD

**Discussion by debjanipaul** [h2oai/wave#1468] [Q&A] [answered]
I am trying to render a county level choropleth animation with plotly express, it works for 100 rows in the dataset but if I load the whole dataset or increase the row values above 100, the app crashes and I see the following error in the command prompt, ' raise ServiceError(f'Request failed (code={res.status_code}): {res.text}')
h2o_wave.core.ServiceError: Request failed (code=413): Request Entity Too Large'.

**Comment by lo5** [comment, marked as answer]
Launch the wave server with a larger -max-request-size. The default is 5 megabytes, which is intentionally low, so you might be hitting the limit there. https://wave.h2o.ai/docs/configuration
